automation/Extension/webext-instrumentation/src/lib/http-post-parser.ts

```diff
@@ -60,8 +60,7 @@ export class HttpPostParser {
     }
     if (requestBody.formData) {
       return {
-        // TODO: requestBody.formData should probably be transformed into another format
-        post_body: requestBody.formData,
+        post_body: escapeString(JSON.stringify(requestBody.formData)),
```

Comment:
PS For a future clean-up we should probably rename this method to escapeStringAndUtf8Encode to clarify what is going on here.